src/main/java/com/redhat/pantheon/conf/LocalFileManagementService.java

```diff
@@ -38,54 +40,77 @@ public void activate() throws IOException {
     }
 
     public void initializeTemplateDirectories() throws IOException {
-        Enumeration<URL> urls = FrameworkUtil.getBundle(LocalFileManagementService.class)
-                .findEntries("apps/pantheon/templates/haml/html5", "*", false);
+//        Enumeration<URL> urls = FrameworkUtil.getBundle(LocalFileManagementService.class)
+//                .findEntries("apps/pantheon/templates/haml/html5", "*", false);
+        Optional<Enumeration<URL>> urls = Optional.ofNullable(FrameworkUtil.getBundle(LocalFileManagementService.class)
+                .findEntries("apps/pantheon/templates/haml/html5", "*", false));
         Path p = Files.createTempDirectory("templates");
 
         log.info("Initializing template directories at " + p.toString());
 
-        templateDirectory = Optional.of(p.toFile());
-
-        while (urls.hasMoreElements()) {
-            URL url = urls.nextElement();
-            String filename = url.toString();
-            filename = filename.substring(filename.lastIndexOf("/") + 1);
-
-            File f = new File(templateDirectory.get(), filename);
-            f.deleteOnExit();
-
-            InputStream is = url.openConnection().getInputStream();
-
-            FileUtils.copyInputStreamToFile(is, f);
+//        if (urls != null && urls.hasMoreElements()) {
+//            templateDirectory = Optional.of(p.toFile());
+//        }
+        if (urls.orElse(Collections.emptyEnumeration()).hasMoreElements()) {
+            templateDirectory = Optional.of(p.toFile());
         }
+
+//        while (urls != null && urls.hasMoreElements()) {
+//            URL url = urls.nextElement();
+//            String filename = url.toString();
+//            filename = filename.substring(filename.lastIndexOf("/") + 1);
+//
+//            File f = new File(templateDirectory.get(), filename);
+//            f.deleteOnExit();
+//
+//            InputStream is = url.openConnection().getInputStream();
+//
+//            FileUtils.copyInputStreamToFile(is, f);
+//        }
```

Comment:
Lets get rid of commented out code.

Reply:
I left it in place as a good example between old-school and new-school, but ok, I removed it. I understand we don't want to clutter our codebase. I also took the opportunity to change the if-statement in the middle too, let me know if you have thoughts on lambdas in the form of "a -> (something unrelated to 'a')"